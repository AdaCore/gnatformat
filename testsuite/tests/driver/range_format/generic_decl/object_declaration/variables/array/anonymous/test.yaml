description: ObjDecl formatting from line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 69",
    "-EL 70",
    "--pipe",
  ]
