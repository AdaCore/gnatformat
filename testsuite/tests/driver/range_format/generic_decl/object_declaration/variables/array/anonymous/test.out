test.ads(<ObjectDecl ["Color_Table"] test.ads:69:4-69:43>) - 69:1-69:43
^
   Color_Table : array (1 .. Max) of Color;$

